modules/backendscheduler/backendscheduler.go

```diff
@@ -290,11 +291,14 @@ func (s *BackendScheduler) UpdateJob(ctx context.Context, req *tempopb.UpdateJob
 		var (
 			metas     = s.store.BlockMetas(j.Tenant())
 			oldBlocks []*backend.BlockMeta
+			bb        uuid.UUID
 		)
 
 		for _, b := range j.GetCompactionInput() {
+			bb = (uuid.UUID)(backend.MustParse(b))
```

Comment:
Are there concerns over a panic here?

Reply:
Calling parse and checking the error seems more reliable.

Reply:
I wasn't concerned until you asked. :)

I believe this won't be an issue, because we convert the blockIDs to strings [here](https://github.com/zalegrala/tempo/blob/4c7d3f91f2bed4eab301a5d336c7cfe5cfa64e2e/modules/backendscheduler/provider/compaction.go#L197) which is then converted to proto [here](https://github.com/zalegrala/tempo/blob/4c7d3f91f2bed4eab301a5d336c7cfe5cfa64e2e/modules/backendscheduler/provider/compaction.go#L182).  Then it's json in/out to the backend, and proto between the components.

To be cautious we could convert this to a Parse call and log the resulting error if we had one.  What do you think?